service/src/LevelZeroDevicePool.hpp

```diff
@@ -24,7 +24,70 @@ namespace geopm
             /// @return Number of LevelZero GPUs.
             /// @param [in] domain The GEOPM domain type being targeted
             virtual int num_gpu(int domain_type) const = 0;
-
+            // RAS SIGNAL FUNCTIONS
```

Comment:
Please move these to the bottom of the public section.

Reply:
Noted. Fixed.